src/style.css

```diff
@@ -1 +1,72 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", serif;
```

Comment:
"Roboto is sans-serif", also you may add one certain sans-serif font (e.i Helvetica)